Archisinal.md

```diff
@@ -0,0 +1,310 @@
+
+**Oleh Kulykovskyi** - front-end developer with 4.5 years of work experience including 3.5 years working with React & Next.JS, mostly in TypeScript language. Proficient with numerous business-grade tools like Redux, GraphQL, Git, Github, animation libraries (GSAP, Framer), MUI, Jest etc. Implemented a broad range of web solutions from landing pages focused on visuals to complex system of business-grade interconnected apps. Created and implemented a web3 platform for a project in Polkadot ecosystem - [Asylum space](https://github.com/w3f/Grants-Program/blob/master/applications/asylum.md). Developed an npm package as a solution for integrating blockchain into React. Tech stack: JS, React.js, Next.js, web3.js, polkadot.js, Typescript.
+
+**Artem Smolyarenko** - UI/UX designer with over 3 years experience. He specializes in brand development in all aspects and have an excellent understanding of typography, color psychology, as well as grid, web and mobile principles. Artem worked on different types of products with startups and corporate teams from around the world. Skills: UI/UX design, product design, web and mobile app design. Tech stack & tools: Figma, Adobe Photoshop, Adobe Illustrator, Adobe XD.
+
+### Team Code Repos
+
+- **Archisinal** [https://github.com/Archisinal](https://github.com/Archisinal)
+- **Danylo Semirazov** - [github.com/sventime](https://github.com/sventime), [gitlab.com/asylum-space](https://gitlab.com/asylum-space)
+- **Oleh Kulykovskyi**  - [github.com/blindbeat](http://github.com/blindbeat), [gitlab.com/asylum-space](https://gitlab.com/asylum-space)
+
+### Team LinkedIn Profiles (if available)
+
+- **Tom Humber** https://www.linkedin.com/in/tom-humber-b65888118
+- **Danylo Semirazov** - [linkedin.com/in/danylosemirazov/](https://www.linkedin.com/in/danylosemirazov/)
+- **Oleh KulyKovskyi**  - [linkedin.com/in/oleh-kulykovskyi-329523234/](http://www.linkedin.com/in/oleh-kulykovskyi-329523234/)
+
+
+## Development Status :open_book:
+
+- [Medium](https://medium.com/@archisinal)
+- [Twitter](https://twitter.com/archisinal)
+- [Instagram](https://www.instagram.com/archisinal.io/)
+
+## Development Roadmap :nut_and_bolt: ##
+
+### Overview
+
+The development roadmap for Archisinal includes multiple milestones to ensure the successful launch and operation of the platform.
+
+### **Overview**
+
+- **Total Estimated Duration:**
+    - MVP: 2.5 months
+    - Post-MVP: 9 months
+    - Post-Launch: 10 months
+- **Full-Time Equivalent (FTE): 6**
+- **Total Costs:**
+    - MVP: $60,000
+
+## MVP (brief discussion)
+
+### **Milestone 1: Smart Contract Development**
+
+- **Estimated duration:** 1 month
+- **FTE:** 3
+- **Costs:** 24,000 USD
+
+**Overview:**
+
+- Develop smart contracts for NFT minting, buying, selling, and royalty distribution
```

Comment:
Here you propose to create your own NFT platform. However, building infrastructure requires a lot of time, and the maintenance is even more costly. Given the mission of the team is to create a marketplace dedicated for AEC, it looks to me a better idea is to build your project on top of some existing NFT projects (for example, [RMRK ink!](https://github.com/rmrk-team/rmrk-ink)). In this way, you can focus on your own market and domain knowledge rather than building an infrastructure-heavy project.

The benefit of using a shared infra is huge:

1. You save a good amount of time to build your own unique features and UX
2. You can utilize all the future innovation & updates from the infra project
3. You can get additional traffic from the infra community
4. You will build a strong relationship in the general ink! ecosystem instead of becoming a competitor of other infra team

Have you considered this option?

Reply:
Hi!

Thanks for taking the time to review the application. I will let Danil weigh in with his past experiences regarding using shared infrastructure, but I can add my own thoughts from a general perspective.

The NFT marketplace is the MVP of a project which will eventually morph into a dispute and arbitration platform for professionals in the construction and architecture industry. The technical requirements for this platform will be continuously morphing, integrating DAO functionality, DIDs and payment automation- this will require connection to Polkadot's various parachains. For these reasons, I feel it important that Archisinal is built from scratch with it's own code base and not reliant on other protocol standards.

Furthermore, I will be speaking at the [Construction Blockchain Consortium](https://www.constructionblockchain.org/) in Berlin in August, and would like to use it as an opportunity to highlight the benefits of building and hosting a Dapp on Astar from scratch. The community/target audience for this project will be commercial industry professionals as opposed to web3 native NFT, so I would like to build a sovereign platform which is more tailored to that demographic.

Reply:
Hi @h4x3rotab! 

I appreciate your valid remarks. There is no great variety of NFT infrastructure for ink! nowadays, but RMRK is definitely the most prominent and suitable. Let me add my thoughts from a tech perspective on why it may be not the best choice for our project:

1. The RMRK standard far exceeds our requirements. Its main features such as nesting, equippability, and multi-resources, are not relevant to our use cases. Our needs align more closely with the basic NFT features provided by PSP34, which RMRK ink! actually reuses in their implementation.
2. Currently, there doesn't seem to be an RMRK ink! marketplace implementation. There is for sure rmrk-pallet for substrate, but not for ink!

Taking the above into account the integration with RMRK at this point, especially for MVP can make more difficulties than benefits. I can tell this from my previous experience with [Asylum](https://github.com/w3f/Grants-Program/blob/master/applications/asylum.md).
For sure, we won't build everything from scratch and we intend to leverage tools such as OpenBrush, Swanky, and Typechain.

Reply:
Good feedback. We discussed internally and agree that the ecosystem is young. This is a good time for you guys to explore and address your unique business goals.